src/rngs/adapter/read.rs

```diff
@@ -103,6 +103,8 @@ impl std::error::Error for ReadError {
 
 #[cfg(test)]
 mod test {
+    use std::{vec, println};
```

Comment:
Vector is not needed here either, since `ReadRng` can work with slices.

Reply:
Sure, I'll make it use arrays instead.